src/client/features/view/components/menu/index.js

```diff
@@ -78,40 +117,78 @@ class Menu extends React.Component {
+      return (
+        h('div', {
+          key: index,
+          className: classNames('tool-button', 'tool-button-sidebar', this.props.activeMenu === button ? 'tool-button-active' : ''),
+          onClick: () => this.changeMenu(button),
+          title: toolButtons[button]
+        }, [
+            h('i.material-icons', button)
+          ])
+      );
+    });
+
     return (
-      h('div.menu-bar', [
+      h('div', {
+        className: classNames('menu-bar', this.props.activeMenu ? 'menu-bar-margin' : '')
+      }, [
         h('div.menu-bar-inner-container', [
           h('div.pc-logo-container', [
-            h(Link, {to: {pathname: '/search'}}, [
+            h(Link, { to: { pathname: '/search' } }, [
               h('img', {
                 src: '/img/icon.png'
               })
             ])
           ]),
           h('div.title-container', [
             h('h4', `${this.props.name} | ${this.props.datasource}`)
-          ]),
-          h('div.search-nodes', {
-            onChange : e => searchNodes(e.target.value, this.props.cy),
-            title: 'Search for Nodes'
+          ])
+        ]),
+        h('div.view-toolbar', toolButtonEls.concat([
+          h('div.tool-button', {
+            onClick: () => this.toggleExpansion(),
+            title: 'Expand/collapse complexes'//`${this.state.complexesExpanded ? 'Collapse' : 'Expand'} complexes`
+          }, [h('i.material-icons', this.state.complexesExpanded ? 'select_all' : 'settings_overscan')]),
+          h('div', {
+            className: classNames('tool-button', this.state.dropdownOpen ? 'tool-button-active' : ''),
+            onClick: () => this.setState({ dropdownOpen: !this.state.dropdownOpen }),
+            title: 'Rearrange entities'
+          }, [h('i.material-icons', 'shuffle')]),
+          h('div', {
+            className: classNames('layout-dropdown', this.state.dropdownOpen ? 'layout-dropdown-open' : '')
           }, [
-            h('div.view-search-bar', [h('input.view-search', {type : 'text', placeholder: 'Entity Search'})])
+            h(Dropdown, {
+              value: this.state.selectedLayout,
+              onChange: value => this.performLayout(value)
+            }, layoutItems)
           ]),
-          h('div.layout-dropdown-button', {
-            onClick: () => this.setState({dropdownOpen: !this.state.dropdownOpen}),
-            title: 'Rearrange the entities on screen'
+          h('div', {
+            className: classNames('tool-button', this.state.searchOpen ? 'tool-button-active' : ''),
+            onClick: () => {
+              !this.state.searchOpen || this.clearSearchBox();
+              this.setState({ searchOpen: !this.state.searchOpen });
+            },
+            title: 'Search entities'
+          }, [h('i.material-icons', 'search')]),
+          h('div', {
+            className: classNames('search-nodes', this.state.searchOpen ? 'search-nodes-open' : ''),
```

Comment:
It's probably cleaner to use with an object, e.g. `classNames({ 'foo-bar': true })`.

Reply:
Huh. I didn't know it could be done like that. I changed all examples I could find of it.